<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>854</width>
    <height>519</height>
   </rect>
  </property>
  <property name="cursor">
   <cursorShape>ArrowCursor</cursorShape>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="1">
     <widget class="QWidget" name="widget" native="true">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>1600</width>
        <height>1600</height>
       </size>
      </property>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout">
         <property name="spacing">
          <number>0</number>
         </property>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>238</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="label_2">
           <property name="frameShape">
            <enum>QFrame::NoFrame</enum>
           </property>
           <property name="text">
            <string>Jemmy</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Minimum</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>13</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_4">
           <property name="minimumSize">
            <size>
             <width>50</width>
             <height>50</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>50</width>
             <height>50</height>
            </size>
           </property>
           <property name="autoFillBackground">
            <bool>false</bool>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>../../../../Downloads/avatar.jpeg</normaloff>../../../../Downloads/avatar.jpeg</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>50</width>
             <height>50</height>
            </size>
           </property>
           <property name="autoRepeatInterval">
            <number>300</number>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Minimum</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>0</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item row="1" column="0">
        <widget class="QStackedWidget" name="stackedWidget">
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="QWidget" name="home">
          <widget class="QLabel" name="label_4">
           <property name="geometry">
            <rect>
             <x>260</x>
             <y>20</y>
             <width>261</width>
             <height>16</height>
            </rect>
           </property>
           <property name="font">
            <font>
             <family>Academy Engraved LET</family>
             <pointsize>14</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Welcome to Blackjack game</string>
           </property>
          </widget>
          <widget class="QFrame" name="frame">
           <property name="geometry">
            <rect>
             <x>300</x>
             <y>220</y>
             <width>91</width>
             <height>61</height>
            </rect>
           </property>
           <property name="styleSheet">
            <string notr="true">border-radius: 30px;
border-color: rgb(255, 255, 255);
border-width: 2px; 
border-style: solid;</string>
           </property>
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <widget class="QPushButton" name="pushButton_5">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>0</y>
              <width>91</width>
              <height>60</height>
             </rect>
            </property>
            <property name="minimumSize">
             <size>
              <width>90</width>
              <height>60</height>
             </size>
            </property>
            <property name="cursor">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: rgb(58, 32, 32);
border-radius: 30px;
border-color: rgb(255, 255, 255);</string>
            </property>
            <property name="text">
             <string>Play game</string>
            </property>
           </widget>
          </widget>
          <widget class="QWidget" name="">
           <property name="geometry">
            <rect>
             <x>140</x>
             <y>70</y>
             <width>416</width>
             <height>112</height>
            </rect>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QPushButton" name="playAloneBtn">
              <property name="minimumSize">
               <size>
                <width>200</width>
                <height>100</height>
               </size>
              </property>
              <property name="cursor">
               <cursorShape>PointingHandCursor</cursorShape>
              </property>
              <property name="styleSheet">
               <string notr="true">background-color: rgb(58, 32, 32);
border-color: rgb(255, 255, 255);
border-radius: 10px;
</string>
              </property>
              <property name="text">
               <string>Play Alone</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="playGroupBtn">
              <property name="minimumSize">
               <size>
                <width>200</width>
                <height>100</height>
               </size>
              </property>
              <property name="cursor">
               <cursorShape>PointingHandCursor</cursorShape>
              </property>
              <property name="styleSheet">
               <string notr="true">background-color: rgb(58, 32, 32);
border-color: rgb(255, 255, 255);
border-radius: 10px;</string>
              </property>
              <property name="text">
               <string>Play with other player</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
         <widget class="QWidget" name="room">
          <widget class="QPushButton" name="createNewRoom">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>10</y>
             <width>200</width>
             <height>30</height>
            </rect>
           </property>
           <property name="minimumSize">
            <size>
             <width>200</width>
             <height>30</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>200</width>
             <height>30</height>
            </size>
           </property>
           <property name="cursor">
            <cursorShape>PointingHandCursor</cursorShape>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgb(0, 178, 255);
border-radius: 10px;</string>
           </property>
           <property name="text">
            <string>Create new room</string>
           </property>
          </widget>
          <widget class="QPushButton" name="pushButton_2">
           <property name="geometry">
            <rect>
             <x>240</x>
             <y>10</y>
             <width>200</width>
             <height>30</height>
            </rect>
           </property>
           <property name="minimumSize">
            <size>
             <width>200</width>
             <height>30</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>200</width>
             <height>30</height>
            </size>
           </property>
           <property name="cursor">
            <cursorShape>PointingHandCursor</cursorShape>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgb(0, 178, 255);
border-radius: 10px;</string>
           </property>
           <property name="text">
            <string>Join random</string>
           </property>
          </widget>
          <widget class="QWidget" name="">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>60</y>
             <width>731</width>
             <height>361</height>
            </rect>
           </property>
           <layout class="QGridLayout" name="gridLayout_2">
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="spacing">
             <number>0</number>
            </property>
            <item row="3" column="0">
             <widget class="RoomItemWidget" name="widget_5" native="true">
              <property name="styleSheet">
               <string notr="true">background-color: rgb(255, 255, 65);</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="RoomItemWidget" name="widget_6" native="true">
              <property name="styleSheet">
               <string notr="true">background-color: rgb(255, 255, 85);</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="RoomItemWidget" name="widget_7" native="true"/>
            </item>
            <item row="2" column="1">
             <widget class="RoomItemWidget" name="widget_8" native="true"/>
            </item>
            <item row="3" column="1">
             <widget class="RoomItemWidget" name="widget_3" native="true">
              <property name="styleSheet">
               <string notr="true">background-color: rgb(64, 255, 66);</string>
              </property>
             </widget>
            </item>
            <item row="0" column="0">
             <widget class="RoomItemWidget" name="widget_2" native="true">
              <property name="styleSheet">
               <string notr="true">background-color: rgb(255, 4, 255);</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="RoomItemWidget" name="widget_4" native="true">
              <property name="styleSheet">
               <string notr="true">background-color: rgb(255, 255, 67);</string>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="RoomItemWidget" name="widget_9" native="true"/>
            </item>
           </layout>
          </widget>
         </widget>
         <widget class="QWidget" name="invite">
          <widget class="QLabel" name="label_3">
           <property name="geometry">
            <rect>
             <x>300</x>
             <y>190</y>
             <width>58</width>
             <height>16</height>
            </rect>
           </property>
           <property name="text">
            <string>invite</string>
           </property>
          </widget>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QWidget" name="menu_widget" native="true">
      <property name="maximumSize">
       <size>
        <width>141</width>
        <height>400</height>
       </size>
      </property>
      <property name="toolTipDuration">
       <number>-1</number>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(58, 32, 32);</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="spacing">
        <number>-1</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Menu</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <property name="spacing">
          <number>-1</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QPushButton" name="homeBtn">
           <property name="text">
            <string>Home</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="roomBtn">
           <property name="text">
            <string>Room</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="inviteBtn">
           <property name="text">
            <string>Invite</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>300</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>RoomItemWidget</class>
   <extends>QWidget</extends>
   <header>roomitemwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
